### PR TITLE
Add router firmware for LAUNCHXL-CC26X2R1

### PR DESCRIPTION
**Pull request by dzungpv**

### CODE REVIEW

File: router/CC2652R1/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# LAUNCHXL-CC26X2R1 router firmware
+
+- Router firmware for [LAUNCHXL-CC26X2R1](https://www.ti.com/tool/LAUNCHXL-CC26X2R1) base on **zr_light_CC26X2R1_LAUNCHXL_tirtos_ccs** sample for sdk_4_20_00_35.
+- How to use: It will recognize as **DNCKATSW001** in zigbee2mqtt, work with all available channel, just flash and power the board and allow join. 
+To reset the router, press and hold button **BTN-1** for 10 second and release when led blink.
+- Currently no compilation instructions are available for this firmware.
```
Comment (Koenkk) [inline router/CC2652R1/README.md:6]: Please add these (so also firmwares for CC2652RB and CC1352-P2 can be compiled and maintained)

Comment (dzungpv) [reply router/CC2652R1/README.md:6]: I will add it later. Because three devices need 3 project with additional code changes

Comment (Koenkk) [reply router/CC2652R1/README.md:6]: Hello, are you still planning on sharing instructions on how to compile this FW?

Comment (dzungpv) [reply router/CC2652R1/README.md:6]: @Koenkk I could not share the modification to the firmware but only the hex file. Now i still don't have time for the CC2652RB and CC1352-P2

Comment (Koenkk) [reply router/CC2652R1/README.md:6]: What did you change to make it work on all channels and what to make it also reset with btn1?

**Pull request closed without merging**